atomist/integrate/github.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Connect to GitHub
+description:
+keywords:
+---
+
+{% include atomist/disclaimer.md %}
+
+When installed for a GitHub organization, the Atomist GitHub app links
```

Comment:
I wonder if its worth a few more lines talking about what features this enables.